cache/lru_cache.cc

```diff
@@ -366,44 +353,43 @@ Status LRUCacheShard::Insert(const Slice& key, uint32_t hash, void* value,
     // is freed or the lru list is empty
     EvictFromLRU(charge, &last_reference_list);
 
-    if (usage_ - lru_usage_ + charge > capacity_ &&
+    if ((usage_ + charge) > capacity_ &&
```

Comment:
If I understand correctly, even this PR has not been merged, this line should also also be changed to 
`(usage_ + charge) > capacity_`
@elipoz , would you please help to give some insights here?